daml/Marketplace/Clearing/Market/Service.daml

```diff
@@ -16,8 +20,18 @@ template Service
     key (operator, provider, customer) : (Party, Party, Party)
     maintainer key._1
 
-    controller customer can
+    nonconsuming choice RequestFairValues : ContractId FairValueCalculationRequest
```

Comment:
```suggestion
    nonconsuming choice RequestFairValue : ContractId FairValueCalculationRequest
```
Is this a typo?

Reply:
It was not since you are able to use this choice to request `FairValue`s for multiple listings. Now that I'm looking at it, it might make some sense to have multiple choices - `RequestFairValue` for one listing, `RequestFairValues` for multiple listings, and `RequestAllFairValues` for all listings instead of having the `Optional` that doesn't really suggest the actual functionality (`None` meaning all listings)...

Reply:
I split the choice as mentioned above.